plugins/modules/eos_pyateos.py

```diff
@@ -0,0 +1,615 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Federico Olivieri (lvrfrc87@gmail.com)
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = """
+module: eos_pyateos
+version_added: "2.10"
+author: "Federico Olivieri (@Federico87)"
+short_description: Operational status tests on Arista device.
+description:
+    - A snapshot of the operational status of a switch is taken before a
+    config or network change and compare against a second snapshot taken after the change.
+    A diff file is generated in .json format.
+notes:
+    - Tested against EOS 4.18
```

Comment:
All notes and descriptions should consist of (potentially multiple) complete sentences per paragraph.
```suggestion
    - Tested against EOS 4.18.
```